create_release.sh

```diff
@@ -1,7 +1,20 @@
-docker run --rm  -v"$(pwd)":/app  -ti semantic-release  -- semantic-release version
-export pwd=$(pwd)
-export TAG=$(docker run --tty=false --rm  -v$pwd:/app  -i semantic-release -- semantic-release version --noop | grep "Current version: " | cut -d ' ' -f 3 | sed -e "s/\r//")
+
+# if a tag hasn't been passed in as a parameter, create a semantic one
+if [ ! -n "$TAG" ]; then
+    git commit --allow-empty -m "generating a new release"
```

Comment:
i see - this was moved here. it may be necessary regardless if we use custom tags or not